.github/workflows/clossing-issues.yml

```diff
@@ -0,0 +1,21 @@
+name: 'Close Solved issues'
```

Comment:
Since we already have several actions running in this repo, I would add a prefix for the ones related to support, so we can identify them in this view https://github.com/bitnami/charts/actions
```suggestion
name: '[Support] Close Solved issues'
```

Reply:
Fixed